<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Source Code Pro</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>COM and VISA</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>221</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,7,1">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_OSC"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_COM"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_COM">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string>COM</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_OSC">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string>OSC</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_linkOSC">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_linkCOM">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_scan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>扫描</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_COM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_browser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_select_wavefiles_path">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>选择保存路径</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>111</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>电机调试（单位度）</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_mMode">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_mMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>mMode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>vMode</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_vMode">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_pdMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>pdMode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ptMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ptMode</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pdMode">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ptMode">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_motorDebugStop">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>180</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_motorSetZero">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>210</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SET ZERO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_motorBangZero">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>240</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_selectMotor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>MotorX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MotorY0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MotorY1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MotorZ</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>框架（单位毫米）</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_X">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Y">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Zpressure">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-800</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_go_XY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>41</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_X">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Z">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>45</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Zup">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>23</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>压力</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_go_Zup">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_go_Zdown">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_homeZ">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_homeXY">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Y_Max">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MaxY</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_X_Max">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MaxX</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clr_stuck">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ClearStuckProtect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>自动测试程序（单位毫米）</string>
    </property>
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X采样间隔</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y采样间隔</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Y_samp_interval">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_X_samp_interval">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FromX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FromY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X采样点数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y采样点数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_X_sample_num">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y_sample_num">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_sample_num">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>总采样点数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>130</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>351</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Stop">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_sample">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前采样点数  =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ToX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ToY</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FromX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ToY">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FromY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ToX">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DrawRange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>DrawRange</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_OSC">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>111</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>OSC</string>
    </property>
    <widget class="QPushButton" name="pushButton_OSCSave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKitar">
    <property name="title">
     <string>Kitar~</string>
    </property>
   </widget>
   <addaction name="menuKitar"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
